project-utils/seed.js

```diff
@@ -0,0 +1,731 @@
+    helper: '@avoir',
+    minimum: 2
+  },
+  {
+    name: 'dictated but not read',
+    value: 134,
+    logoURL: 'https://unsplash.com/photos/ISL7czxIP-k',
+    helper: '@cafard',
+    minimum: 6
+  },
+  {
+    name: 'die',
+    value: 114,
+    logoURL: 'https://unsplash.com/photos/t1mqA3V3-7g',
+    helper: '@banco',
+    minimum: 1
+  },
+  {
+    name: 'dig out',
+    value: 108,
+    logoURL: 'https://unsplash.com/photos/OJJIaFZOeX4',
+    helper: '@datos',
+    minimum: 2
+  },
+  {
+    name: "don't feed the troll",
+    value: 154,
+    logoURL: 'https://unsplash.com/photos/688Fna1pwOQ',
+    helper: '@blow',
+    minimum: 1
+  },
+  {
+    name: 'drive-by media',
+    value: 185,
+    logoURL: 'https://unsplash.com/photos/JzkgpML_8XI',
+    helper: '@someones',
+    minimum: 5
+  },
+  {
+    name: 'et alibi',
+    value: 149,
+    logoURL: 'https://unsplash.com/photos/fwsvUxNgLRs',
+    helper: '@phone',
+    minimum: 1
+  },
+  {
+    name: 'et uxor',
+    value: 178,
+    logoURL: 'https://unsplash.com/photos/h13Y8vyIXNU',
+    helper: '@bright',
+    minimum: 6
+  },
+  {
+    name: 'fiat lux',
+    value: 136,
+    logoURL: 'https://unsplash.com/photos/YvYBOSiBJE8',
+    helper: '@line',
+    minimum: 6
+  },
+  {
+    name: 'fill in the blank',
+    value: 165,
+    logoURL: 'https://unsplash.com/photos/Cm7oKel-X2Q',
+    helper: '@caerse',
+    minimum: 3
+  },
+  {
+    name: 'gather rosebuds',
+    value: 161,
+    logoURL: 'https://unsplash.com/photos/D76DklsG-5U',
+    helper: '@las',
+    minimum: 2
+  },
+  {
+    name: 'give the lie',
+    value: 125,
+    logoURL: 'https://unsplash.com/photos/eLUegVAjN7s',
+    helper: '@manos',
+    minimum: 2
+  },
+  {
+    name: 'goodbye, cruel world',
+    value: 195,
+    logoURL: 'https://unsplash.com/photos/kmF_Aq8gkp0',
+    helper: '@cargo-200',
+    minimum: 1
+  },
+  {
+    name: 'hand on a plate',
+    value: 199,
+    logoURL: 'https://unsplash.com/photos/yZf1quatKCA',
+    helper: '@carry',
+    minimum: 6
+  },
+  {
+    name: 'hit the nail on the head!',
+    value: 162,
+    logoURL: 'https://unsplash.com/photos/llYg8Ni43fc',
+    helper: '@forward',
+    minimum: 1
+  },
+  {
+    name: 'I beg your pardon',
+    value: 103,
+    logoURL: 'https://unsplash.com/photos/A6S-q3D67Ss',
+    helper: '@over',
+    minimum: 3
+  },
+  {
+    name: 'i go batty for you!',
+    value: 183,
+    logoURL: 'https://unsplash.com/photos/YcfCXxo7rpc',
+    helper: '@causam',
+    minimum: 6
+  }
+];
+
+const seed = () => {
+  return Bundles.findOne({
+    where: { id: 100 },
+  })
+  .then((results) => {
+    if (results === null) {
+      return Bundles.bulkCreate(seeds, {
+        fields: ['name', 'value', 'logoURL', 'helper', 'minimum'],
+      });
+    } else {
+      console.log(`db populated: ${results.name} is the last bundle name`);
+    }
+  });
+};
+
+seed()
+.then(() => {
+  process.exit();
+})
+.catch((err) => {
+  console.log(err);
+});
+
+module.exports = seed;
```

Comment:
Wait did you manually seed your data?! Madlad haha. I have no comment here 😂

Reply:
I used a helper function in another file to grab 100 URLs, phrases, and words, and 100 random values between 100-200 and 1-7, then combined those into objects and saved the objects. Since the data theoretically doesn't have to change content-wise other than when I deploy my 100 images, I wanted a hard copy object with that data for ease of use. Now if I have to recreate the process, at least it's just the seed file that needs writing.